webapp/.gitignore

```diff
@@ -110,3 +110,6 @@ Cargo.lock
 *.pdb
 
 # End of https://www.toptal.com/developers/gitignore/api/flutter,dart,rust
+
+# data directory for webapp
+data
```

Comment:
I thought we were meaning to share the `data` directory with the root of the project. This is not wrong though.

Reply:
Yea, but if you run `cargo run` from the sub folder it will generate this which is not wrong but annoying to have :)